client/my-sites/upgrades/domain-management/dns/email-provider.jsx

```diff
@@ -75,7 +80,10 @@ class Office365 extends Component {
 						<FormButton
 							disabled={ ! isDataValid || this.state.submitting }
 							onClick={ this.onAddDnsRecords }>
-							{ translate( 'Set up Office 365' ) }
+							{ translate(
+								'Set up %(providerName)s',
```

Comment:
Please add a translator comment too clarifying what the providerName is (i.e "Email service, like G Suite, Office365, etc.")